index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/0aca4b41e8.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>UI-Challenge</title>
+  </head>
+  <body>
+    <div id="ui-challenge">
+      <div class="container">
+        <img id="img1" src="/google-dots.png" alt="color_code" />
+        <i class="fa-solid fa-circle-chevron-left"></i>
+        <div class="header">
+          <img
+            src="https://res.cloudinary.com/teepublic/image/private/s--XuYxkWhu--/c_crop,x_10,y_10/c_fit,h_830/c_crop,g_north_west,h_1038,w_1038,x_-108,y_-104/l_upload:v1565806151:production:blanks:vdbwo35fw6qtflw9kezw/fl_layer_apply,g_north_west,x_-219,y_-215/b_rgb:000000/c_limit,f_jpg,h_630,q_90,w_630/v1569026033/production/designs/6035915_0.jpg"
+            alt="logo"
+          />
+          <h2>Aceagency</h2>
+        </div>
```

Comment:
The image was inside the <h2> It should be adjacent and then use CSS for positioning it. Check the CSS for this section to see how.